core/src/main/resources/messages/messages_nl.properties

```diff
@@ -101,6 +101,8 @@ error.no.device.request.present=Er bestaat geen ''new device request'' voor de g
 error.device.request.expired=Het ''new device request'' voor gebruiker met ID ''{0}'' is verlopen
 error.device.request.invalid.password=Het tijdelijke wachtwoord voor het ''new device request'' is ongeldig. Is het verzoek misschien vervangen door een ander?
 error.device.unknown.operating.system=Besturingssysteem ''{0}'' wordt niet ondersteund.
+error.device.invalid.device.name=De apparaatnaam ``{0}`` is niet toegestaan. Een geldige apparaatnaam is niet langer dan {1} tekens en bevat geen ''{2}''.
```

Comment:
The quotes are wrong (`` instead of '').

Reply:
There were a few more of these. Corrected them all.